devel/pkgconf/Makefile

```diff
@@ -0,0 +1,71 @@
+#
+# Copyright (C) 2015-2016 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=pkgconf
+PKG_VERSION:=1.6.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
+PKG_SOURCE_URL:=https://distfiles.dereferenced.org/pkgconf
+PKG_HASH:=61f0b31b0d5ea0e862b454a80c170f57bad47879c0c42bd8de89200ff62ea210
+
+PKG_MAINTAINER:=Rosen Penev <rosenp@gmail.com>
+PKG_LICENSE:=MIT
+PKG_LICENSE_FILES:=1
+
+PKG_INSTALL:=1
+PKG_BUILD_PARALLEL:=1
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libpkgconf
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=libpkgconf
+  URL:=http://pkgconf.org/
+endef
+
+define Package/pkgconf
+  SECTION:=devel
+  CATEGORY:=Development
+  TITLE:=pkgconf
+  URL:=http://pkgconf.org/
+  DEPENDS:=+libpkgconf
+  CONFLICTS:=pkg-config
```

Comment:
Try using PROVIDES instead

Reply:
Hmmm not quite.

So if I replace it with PROVIDES , it fixes the menuconfig entry (no longer indented) but I can select both. This is wrong as both packages install /usr/share/aclocal/pkg.m4

Reply:
CONFLICTS on the pkg-config package works properly...